src/vmm/src/signal_handler.rs

```diff
@@ -534,13 +575,6 @@ mod tests {
 
         let child = thread::spawn(move || {
             extern "C" fn signal_handler(_: c_int, siginfo: *mut siginfo_t, _: *mut c_void) {
```

Comment:
There's also the `test_sigsys_handler` test that needs modifications. We no longer need an additional function in the test for registering a classic signal handler and all that `siginfo_t` to `signalfd_siginfo` transformation logic.

That workaround was likely needed because the kernel did not support signalfd for SIGSYS.